### ISSUE TITLE
Best way to rinse probe in tergazyme after use

### ISSUE THREAD

**Issue by weallen** [cortex-lab/neuropixels#9]
Hi,

I wanted to know if anyone had tips on the best way to wash the probe in tergazyme after use (i.e. how to hold it in the beaker/vehicle for tergazyme). Do you just use a micromanipulator?

Thanks,
Will

**Comment by nsteinme** [comment]
I have a setup that sounds very similar to yours, and also involves taking the probe off the rig - but I do leave the probe and headstage attached to a big rod that is held by the micromanipulators. I carry the whole thing with this rod. I think this makes it reasonably straightforward to handle. 

I think Dan Denman had shown some data once suggesting that over a period of weeks, and a dozen or so recordings, he had seen some subtle signal degredation - but this effect went away when he started using tergazyme to clean the probes. The conclusion there is that tergazyme does help prevent some kind of tissue build-up. I suspect it would only be on a slow timescale, though. I just do it once at the end of the day.

**Comment by weallen** [comment]
Yeah, if this is useful for people, we built something that holds a beaker of tergazyme right where the mouse's head was, so we don't have to remove the probe. The times when I've been responsible for breaking the electrode have generally been when I've been manually handling them, and haven't been able to see exactly where the shank is.

At this point, Nick, what would you say your mean time to breakage is (or mean number of usages), and how has it changed as you've become more experienced with them? I'd say my average right now is pretty low, but it might be useful for people to get a sense of this for planning experiments.

**Comment by nsteinme** [comment]
So far as my records reflect, my last broken probe was 14 months ago. But
the records also say I broke six in the first three months or so that we
had this version of the probe. I think it could be well-modeled as a
poisson process with a mean parameter that falls off exponentially over
time :) So don't worry, it gets better from here...

**Comment by weallen** [comment]
Good to know :-) That sounds  about consistent with my rate... I guess the next question would be: is there some change you made that suddenly caused you to stop breaking them?

If it's useful for other people who read this thread, my attempted solution has been to develop a protocol for doing the experiments that basically just minimizes the times when I can't visually see the electrode under a microscope and where I'm physically holding them/moving them around.

**Comment by nsteinme** [comment]
No, no particular change to stop breaking them, but just as you say - finding ways to handle them in safe/secure ways (or avoid handling them).
